### DISCUSSION TITLE
can i use the code in a minecraft client called matchafi

### DISCUSSION THREAD

**Discussion by ItzPrism** [xCollateral/VulkanMod#298] [General]
title. also works with optifine?

**Comment by sqrvrt** [comment]
how do you think it's going to work with optifine bro it's like to installing a mod on bedrock edition

~also this project doesn't have a license so ig do whatever you want. Like what're they gonna do?  Sue you?~
It's licensed under LGPL, you can read it and decide whether you can use it.

**Reply by Felix14-v2** [reply]
> project doesn't have a license so ig do whatever you want

If a project didn't specified its licence, then it is automatically protected by ARR (All Rights Reserved), and you can't do anything with it except direct use.
